.npmrc

```diff
@@ -0,0 +1 @@
+use-node-version=22.14.0
```

Comment:
[FYI]
Previously, we used 20 versions.

Reply:
Thanks!
Agentica uses node 22. I think we can move on to 22!
I need your opinion! @sunrabbit123 @8471919